pkg/util/hostname/README.md

```diff
@@ -83,27 +90,40 @@ EC2 offers two versions of its metadata service, v1 and v2.
+| none    | true                | true                                    | 1      | os         | none       |
+| v2 only | false               | true                                    | 1      | aws (i-..) | aws (i-..) |
+| v2 only | true                | true                                    | 1      | aws (i-..) | aws (i-..) |
+| v1+v2   | false               | true                                    | 1      | aws (i-..) | aws (i-..) |
+| v1+v2   | true                | true                                    | 1      | aws (i-..) | aws (i-..) |
+| none    | false               | true                                    | 2+     | os         | none       |
+| none    | true                | true                                    | 2+     | os         | none       |
+| v2 only | false               | true                                    | 2+     | os         | none       |
+| v2 only | true                | true                                    | 2+     | os         | none       |
+| v1+v2   | false               | true                                    | 2+     | os         | aws (i-..) |
+| v1+v2   | true                | true                                    | 2+     | os         | aws (i-..) |
+| none    | false               | false                                   | 1      | os         | none       |
+| none    | true                | false                                   | 1      | os         | none       |
+| v2 only | false               | false                                   | 1      | os         | none       |
+| v2 only | true                | false                                   | 1      | aws (i-..) | aws (i-..) |
+| v1+v2   | false               | false                                   | 1      | aws (i-..) | aws (i-..) |
+| v1+v2   | true                | false                                   | 1      | aws (i-..) | aws (i-..) |
+| none    | false               | false                                   | 2+     | os         | none       |
+| none    | true                | false                                   | 2+     | os         | none       |
+| v2 only | false               | false                                   | 2+     | os         | none       |
+| v2 only | true                | false                                   | 2+     | os         | none       |
+| v1+v2   | false               | false                                   | 2+     | os         | aws (i-..) |
+| v1+v2   | true                | false                                   | 2+     | os         | aws (i-..) |
+
+- The first column describes the EC2 IMDS configuration: "none" means IMDS is entirely disabled; "v2 only" means that IMDSv1 is disabled, and "v1+v2" is the default setting with both versions available
+- The second column is the `ec2_prefer_imdsv2` configuration value.
+- The third column is the `ec2_imdsv2_transition_payload_enabled` configuration value.
```

Comment:
I think these two statements are redundant, as the column is named after the configuration value